instrumentation/netty-codec-http/src/main/java/brave/netty/http/HttpNettyAdapter.java

```diff
@@ -0,0 +1,27 @@
+package brave.netty.http;
+
+import brave.http.HttpServerAdapter;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpResponse;
+
+final class HttpNettyAdapter extends HttpServerAdapter<HttpRequest, HttpResponse> {
+  @Override public String method(HttpRequest request) {
+    return request.method().name();
+  }
+
+  @Override public String url(HttpRequest request) {
+    StringBuffer url = new StringBuffer("http://");
```

Comment:
You want StringBuilder

Reply:
Thx ps do you have any suggestions below on how to get the full url from an incoming request?